app/components/search/search.service.js

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
to jest mega slabe podejscie. co jesli bedziesz mial 40 serwisow wideo do obslugi? wrzucisz tu spaghetti code?

zrob prosze to o czym pisalem w mailu:
korzystac z videos, ktore de facto bylyby strategy pattern korzystajacym z vimeo i youtube (do vimeo i youtube zrob osobne serwisy, ktore beda mialy ten sam interfejs)